src/screens/login/Login.tsx

```diff
@@ -55,34 +55,34 @@ const Login: React.FunctionComponent<Login> = (props) => {
       });
   };
 
-  const LoginGoogle = async () => {
-    signInWithPopup(auth, new GoogleAuthProvider())
-      .then(async (res) => {
-        // localStorage.setItem("user", JSON.stringify(res.user));
-        console.log("Getting user data");
-        let email: string;
-        if (res.user.email) {
-          email = res.user.email.toString();
-        } else {
-          throw new Error();
-        }
-        let userData = await getDoc(doc(db, "user-profile", email));
-        if (userData.exists()) {
-          console.log("Document data:", userData.data());
-          // Check if first name is set, if not, navigate to user profile
-          if (userData.data().firstName) {
-            navigate("/home");
-          } else {
-            navigate("/profile");
-          }
-        } else {
-          navigate("/profile");
-        }
-      })
-      .catch((e) => {
-        console.error(e);
-      });
-  };
+  // const LoginGoogle = async () => {
+  //   signInWithPopup(auth, new GoogleAuthProvider())
+  //     .then(async (res) => {
+  //       // localStorage.setItem("user", JSON.stringify(res.user));
+  //       console.log("Getting user data");
+  //       let email: string;
+  //       if (res.user.email) {
+  //         email = res.user.email.toString();
+  //       } else {
+  //         throw new Error();
+  //       }
+  //       let userData = await getDoc(doc(db, "user-profile", email));
+  //       if (userData.exists()) {
+  //         console.log("Document data:", userData.data());
+  //         // Check if first name is set, if not, navigate to user profile
+  //         if (userData.data().firstName) {
+  //           navigate("/home");
+  //         } else {
+  //           navigate("/profile");
+  //         }
+  //       } else {
+  //         navigate("/profile");
+  //       }
+  //     })
+  //     .catch((e) => {
+  //       console.error(e);
+  //     });
+  // };
```

Comment:
Why did you remove this?

Reply:
The design removed 'login with google' button